vmdb/app/models/orchestration_template.rb

```diff
@@ -90,6 +93,16 @@ def validate_content(manager = nil)
     "No #{ui_lookup(:model => 'ExtManagementSystem').downcase} is capable to validate the template"
   end
 
+  def validate_format
+    "Cannot validate format because template type is unknown"
```

Comment:
I agree with @blomquisg that this should follow the `NotImplementedError` from vm_or_template.